dali/python/nvidia/dali/ops.py

```diff
@@ -548,6 +586,35 @@ def __init__(self, function, num_outputs=1, device='cpu', synchronize_stream=Tru
                                                      synchronize_stream=synchronize_stream,
                                                      **kwargs)
 
+def _load_arithm_ops():
+    arithm_op_names = ["ArithmeticGenericOp"]
+    for op_name in arithm_op_names:
+      if not hasattr(sys.modules[__name__], op_name):
+          setattr(sys.modules[__name__], op_name,
+                  python_op_factory(op_name, op_device = "cpu"))
+
+_load_arithm_ops()
+
+def _choose_device(inputs):
+    result_device = "cpu"
+    for input in inputs:
+        if input.device == "gpu":
+            result_device = "gpu"
+    return result_device
+
+def _arithm_op(name, *inputs):
+        input_desc = ""
+        for i, input in enumerate(inputs):
+            input_desc += "&" + str(i)
```

Comment:
Constants?

Reply:
This is based on master, and constants are in #1391 which is not merger yet. I will add support for constants in a follow-up after this and #1391 are merged.